### PR TITLE
Implementation of loadflow results completion (flows computation based on voltages)

### PR DESCRIPTION
**Pull request by massimo-ferraro**

### CODE REVIEW

File: loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowParametersTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import com.powsybl.commons.config.MapModuleConfig;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.eu>
+ */
+public class ResultsCompletionLoadFlowParametersTest {
+
+    InMemoryPlatformConfig platformConfig;
+    FileSystem fileSystem;
```
Comment (mathbagu) [inline loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowParametersTest.java]: Add modifier: private

File: loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowPostProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import com.powsybl.commons.config.InMemoryPlatformConfig;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.eu>
+ */
+public class ResultsCompletionLoadFlowPostProcessorTest extends AbstractResultsCompletionLoadFlowTest {
+
+    InMemoryPlatformConfig platformConfig;
+    FileSystem fileSystem;
```
Comment (mathbagu) [inline loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowPostProcessorTest.java]: Add modifier

File: loadflow/pom.xml [hunk 3]
```diff
@@ -26,6 +26,7 @@
     <modules>
         <module>loadflow-api</module>
         <module>loadflow-validation</module>
+        <module>results-completion-loadflow</module>
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply loadflow/pom.xml]: Done

File: loadflow/results-completion-loadflow/src/main/java/com/powsybl/loadflow/ResultsCompletionLoadFlow.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.eu>
+ */
+public class ResultsCompletionLoadFlow implements LoadFlow {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResultsCompletionLoadFlow.class);
+
+    Network network;
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply loadflow/results-completion-loadflow/src/main/java/com/powsybl/loadflow/ResultsCompletionLoadFlow.java]: Field removed in new version

File: loadflow/results-completion-loadflow/src/main/java/com/powsybl/loadflow/ResultsCompletionLoadFlow.java [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    }
+
+    @Override
+    public String getName() {
+        return "Results Completion LoadFlow";
+    }
+
+    @Override
+    public String getVersion() {
+        return "1.0.0";
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply loadflow/results-completion-loadflow/src/main/java/com/powsybl/loadflow/ResultsCompletionLoadFlow.java]: Removed in new version

File: loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowParametersTest.java (hunk 1 above)
Comment (geofjamg) [reply loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowParametersTest.java]: Done

File: loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowPostProcessorTest.java (hunk 2 above)
Comment (geofjamg) [reply loadflow/results-completion-loadflow/src/test/java/com/powsybl/loadflow/ResultsCompletionLoadFlowPostProcessorTest.java]: Done

**Pull request merged**